README.md

```diff
@@ -56,6 +69,8 @@ Convert the following create method in Mongoose to ActiveRecord.
 ```
 
 ```rb
+@author = Author.create(name: "Name")
```

Comment:
In your answer, an author is created with the name "Name." You need to grab this value from `params` instead.

`@author = Author.create!(name: params[:name])`